src/components/App/index.jsx

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import jsPDF from "jspdf";
 import 'svg2pdf.js';
 import './App.css';
-import TarsiaGrid from './cTarsiaGrid';
-import Questions from './cQuestionAnswer'
-import hexGrid from './configGrids';
-import PrintSvgDiv from './cPrintSvgDiv'
+import TarsiaGrid from '../TarsiaGrid';
+import Questions from '../QuestionAnswer'
+import PrintableSVGDiv from '../PrintableSVGDiv'
```

Comment:
Renamed the components to follow pascal casing naming convention and moved them to a components directory